### DISCUSSION TITLE
Why is "@next/next/no-html-link-for-pages" turned off by default in the example?

### DISCUSSION THREAD

**Discussion by aitaro** [vercel/turborepo#5499] [Help] [answered]
In my next.js project using turbo, I wanted to detect "no-html-link-for-pages", but I found that it does not cause a lint error.

It appears that linting is turned off even in the official example. Why is this, and is there any workaround?

Relevant Location: https://github.com/vercel/turbo/blob/main/examples/basic/packages/eslint-config-custom/index.js#L4
PR that turns it off: https://github.com/vercel/turbo/pull/419

**Comment by anthonyshew** [comment, marked as answer]
We've updated our examples recently and this should no longer be a problem. If you're still seeing something wrong, please open an issue. 🙏
